test/models/ui/validation/create_account_validation_test.dart

```diff
@@ -35,6 +35,55 @@ void main() {
       );
 
       expect(createAccount, createAccountCopy);
+      expect(createAccount == createAccountCopy, true);
```

Comment:
I think that this line is a duplication of the line right above it .

Reply:
Fixed with [5141c90](https://github.com/ProximaEPFL/proxima/pull/358/commits/5141c90560386380b3d26de95338fdc3622b7d00)

Reply:
Fixed with [5141c90](https://github.com/ProximaEPFL/proxima/pull/358/commits/5141c90560386380b3d26de95338fdc3622b7d00)